static/phyui/manual_clustering.js

```diff
@@ -104,10 +104,25 @@ define(function (require) {
 
     events.on('kernel_connected.Kernel', function(){
         console.log('session done');
-        myhack.create_result_cell('#placeholder1', "from phyui.session import start_manual_clustering; session = start_manual_clustering('/home/ctaf/src/cortex/data/test_hybrid_120sec.kwik', backend='ipynb_webgl')");
-        myhack.create_result_cell('#placeholder2', "import vispy; \
-                                                    vispy.app.use_app('ipynb_webgl'); \
-                                                    w = session.show_waveforms()");
+        myhack.create_result_cell('#placeholder1', "from phyui.session import start_manual_clustering; \
+                                                    session = start_manual_clustering('/home/ctaf/src/cortex/data/test_hybrid_120sec.kwik', backend='ipynb_webgl')");
+        var cc = myhack.create_result_cell('#placeholder2', "import vispy; \
+                                                             vispy.app.use_app('ipynb_webgl'); \
+                                                             w = session.show_waveforms()");
```

Comment:
* hard-coded paths
* mixing Python code with JavaScript

A possible alternative approach might be to use exclusively Comms for communicating events between Python and JavaScript. So you send a "start_manual_clustering" event with comm.send(), and Python does its stuff. The "kernel.send_cell_code" thing would be reserved to the notebook itself, not the web app.